benchmarks/NodeClassification/models/gcn2.py

```diff
@@ -0,0 +1,101 @@
+"""
+GCNII model in GLI.
+
+References:
+https://github.com/dmlc/dgl
+Pull request #5008
+"""
+
+import math
+from torch import nn
+import torch.nn.functional as F
+from dgl.sparse import from_coo, diag, identity
+
+
+class GCNIIConvolution(nn.Module):
+    """GCNII Conv."""
+
+    def __init__(self, in_size, out_size):
+        """Init."""
+        super().__init__()
+        self.out_size = out_size
+        self.weight = nn.Linear(in_size, out_size, bias=False)
+
+    ###########################################################################
+    # (HIGHLIGHT) Take the advantage of DGL sparse APIs to implement the GCNII
+    # forward process.
+    ###########################################################################
+    def forward(self, a_norm, h, h0, lamda, alpha, index):
+        """Forward."""
+        beta = math.log(lamda / index + 1)
+
+        # Multiply a sparse matrix by a dense matrix.
+        h = a_norm @ h
+        h = (1 - alpha) * h + alpha * h0
+        h = (1 - beta) * h + beta * self.weight(h)
+        return h
+
+
+class GCNII(nn.Module):
+    """GCNII network."""
+
+    def __init__(
+        self,
+        g,
+        in_size,
+        out_size,
+        hidden_size,
+        n_layers,
+        lambda_,
+        alpha,
+        dropout=0.5,
+    ):
+        """Init."""
+        super().__init__()
+        self.hidden_size = hidden_size
+        self.n_layers = n_layers
+        self.lambda_ = lambda_
+        self.alpha = alpha
+
+        # The GCNII model.
+        self.layers = nn.ModuleList()
+        self.layers.append(nn.Linear(in_size, hidden_size))
+        for _ in range(n_layers):
+            self.layers.append(GCNIIConvolution(hidden_size, hidden_size))
+        self.layers.append(nn.Linear(hidden_size, out_size))
+
+        self.activation = nn.ReLU()
+        self.dropout = dropout
+
+        # Create the adjacency matrix of graph.
+        src, dst = g.edges()
+        n = g.num_nodes()
+        a = from_coo(dst, src, shape=(n, n))
+
+        #####################################################
+        # (HIGHLIGHT) Compute the symmetrically normalized adjacency matrix
+        # with Sparse Matrix API
+        #####################################################
+        i = identity(a.shape)
```

Comment:
The adjacency or identity matrix symbol can be preserved since it is quite standard to let them be A or I.

Reply:
The variables with uppercase would make pylint check fails, so I change them all to lowercase.

Reply:
> The adjacency or identity matrix symbol can be preserved since it is quite standard to let them be A or I.

The variables with uppercase would make pylint check fails, so I change them all to lowercase.

Reply:
Since Pylint requires all variables to be snake case, you might want to name the matrixs as `i_mat` or `a_mat` for better readability.

Reply:
Sounds good.